content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -52,6 +52,13 @@ The upgrade workflow at high level is the following:
 
 <!-- steps -->
 
+## Changing the package repository
+
+If you're using the Kubernetes community-owned repositories, you need to change
```

Comment:
nit: at some point, you can economize on the number of words by switching to just "community repositories". (But maybe I'm biased after reading all docs at once)  
```suggestion
If you're using the community repositories, you need to change
```

Reply:
We like to be verbose. For example, it helps readers who don't speak English natively, and who don't know Kubernetes very well.

Even so, I'm -0 on this one (I don't object!)

Reply:
I agree with @sftim here. Additionally, it might make it easier for users who might not have enough context on how our repositories work.